src/js/pages/services/DeploymentsTab.js

```diff
@@ -228,7 +228,7 @@ class DeploymentsTab extends mixin(StoreMixin) {
         linkText = linkText + ' & Remove';
       }
       return (
-        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover"
+        <a className="deployment-rollback button button-link button-danger table-display-on-row-hover pull-right"
```

Comment:
I wasn't aware of this class, but I believe it is legacy. I think we should stay consistent with what we do other tables and use `text-aling-right` on the `td`, like here: https://github.com/dcos/dcos-ui/blob/master/src/js/components/PackagesTable.js#L51